### PR TITLE
fix: info logo issue

### PR DESCRIPTION
**Pull request by Chef-Yogi**
---

## PR-Codex overview
The focus of this PR is to export `chainName` in `utils.ts` and update `CurrencyLogo` to dynamically generate image paths based on the chain name.

### Detailed summary
- Export `chainName` in `CurrencyLogo/utils.ts`
- Add `chainNameToPath` function to dynamically generate image paths
- Update `CurrencyLogo` to use `chainNameToPath` for image path generation

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/Info/components/CurrencyLogo/index.tsx [hunk 1]
```diff
@@ -16,6 +16,17 @@ const StyledLogo = styled(TokenLogo)<{ size: string }>`
...
+  ARB: 'arbitrum',
+  POLYGON_ZKEVM: 'polygon-zkevm',
+}
+
+const chainNameToPath = (chainName: MultiChainName) => {
+  if (chainName === 'BSC') return ''
+  if (CHAIN_NAME_TO_PATH[chainName]) return `${CHAIN_NAME_TO_PATH[chainName]}/`
+  return `${chainName.toLowerCase()}/`
+}
+
```
Comment (ChefJerry) [inline apps/web/src/views/Info/components/CurrencyLogo/index.tsx:29]: duplicated with these one
https://github.com/pancakeswap/pancake-frontend/blob/3d45a094ad6707b7902e1e4e2e9c67b67322e63c/packages/widgets-internal/components/CurrencyLogo/utils.ts#L43-L64

Comment (Chef-Yogi) [reply apps/web/src/views/Info/components/CurrencyLogo/index.tsx:29]: ok, let me use this config

Comment (Chef-Yogi) [reply apps/web/src/views/Info/components/CurrencyLogo/index.tsx]: Done

**Review by ChefJerry — DISMISSED** [review]

**Comment by ChefMomota** [comment]
Fix the web build issue

**Comment by Chef-Yogi** [comment]
fixed

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**